api/main.py

```diff
@@ -117,7 +119,7 @@ async def create_new_chat(
     repeat_last_n: int = 64,
     repeat_penalty: float = 1.3,
     init_prompt: str = "Below is an instruction that describes a task. Write a response that appropriately completes the request. The response must be accurate, concise and evidence-based whenever possible. A complete answer is always ended by [end of text].",
-    n_threads: int = 4,
+    n_threads: int = THREADS / 2,
```

Comment:
This should be `n_threads = THREADS`, in case it's 1/2

Reply:
There's a chance the user wants to use all threads available, which this would limit.

Reply:
@nsarrazin thoughts on this?

Reply:
I think it's fine, it's only shown in the API anyway. This doesn't update the front-end  value which is still 4... But still a good idea. 

I'm not sure why it should be threads/2 though. `llama.cpp` by default uses the total number of threads available so I think we might as well just copy what they do.

Reply:
Could there be anywhere else that could change it at the front-end? I saw that there was an n_threads that sets the default value in models.py, but that should get overwritten in main.py?